apps/frontend/src/pages/not-found/not-found.tsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom";
+
+import styles from "./styles.module.css";
+
+const NotFound = (): JSX.Element => {
+	return (
+		<main className={styles["container"]}>
+			<h1 className={styles["title"]}>404</h1>
+			<h3 className={styles["subtitle"]}>Something went wrong</h3>
+			<p className={styles["text"]}>
+				Sorry, we can’t find the page you’re looking for.
+			</p>
+			<Link className={styles["link"]} to="/">
```

Comment:
This should be a common button component. Let's pass href parameter in it, and in case it's present render NavLink

Reply:
I edit the button component to take a href parameter